nselib/http.lua

```diff
@@ -1630,6 +1638,7 @@ end
 -- @return A response table, see module documentation for description.
 -- @see http.generic_request
 function get(host, port, path, options)
+  options.cookies = stdnse.get_script_args("http.cookiejar") or nil
```

Comment:
What if options is nil? This will throw an exception as options is nil. Another thing is that this will probably introduce undesired behavior. For example, here you are changing internally the value of the header that some scripts might have worked on and are sending internally. You will be overwriting that value for them. I think the best action is to use the given value if exists.

Reply:
Yeah! I see your point. I will check for nil options, and only if options.cookies is nil, will parse the argument. That should be correct, right?